dd-sdk-android-core/src/test/kotlin/com/datadog/android/utils/forge/Configurator.kt

```diff
@@ -43,6 +43,27 @@ internal class Configurator :
         forge.addFactory(ProcessInfoForgeryFactory())
         forge.addFactory(DeviceInfoForgeryFactory())
         forge.addFactory(DatadogContextForgeryFactory())
+        forge.addFactory(DataUploadConfigurationForgeryFactory())
+
+        // UploadStatus
+        forge.addFactory(HttpRedirectStatusForgeryFactory())
+        forge.addFactory(HttpClientRateLimitingStatusForgeryFactory())
+        forge.addFactory(HttpClientErrorForgeryFactory())
+        forge.addFactory(HttpServerErrorForgeryFactory())
+        forge.addFactory(InvalidTokenErrorStatusForgeryFactory())
+        forge.addFactory(NetworkErrorStatusForgeryFactory())
+        forge.addFactory(RequestCreationErrorStatusForgeryFactory())
+        forge.addFactory(SuccessStatusForgeryFactory())
+        forge.addFactory(UnknownErrorStatusForgeryFactory())
+        forge.addFactory(UnknownStatusForgeryFactory())
```

Comment:
cannot it all be in a single `UploadStatus` factory?

Reply:
well I would have loved that too but unfortunately sometimes I need specific `UploadStatus` types in my tests.